npm/ng-packs/packages/theme-shared/src/lib/components/internet-connection-status/internet-connection-status.component.ts

```diff
@@ -42,7 +41,7 @@ import { InternetConnectionService, LocalizationModule } from '@abp/ng.core';
     }
   `]
 })
-export class  InternetConnectionStatusComponent{
-  internetConnectionService = inject(InternetConnectionService)
-  isOffline$ = this.internetConnectionService.networkStatus$.pipe(map((val) => !val));
+export class InternetConnectionStatusComponent{
+  internetConnectionService = inject(InternetConnectionService);
+  isOnline = this.internetConnectionService.networkStatus

```

Comment:
you can use signal compute or converting opposite of value.  it just a note. it works like that.